packages/cozy-harvest-lib/src/components/DeleteAccountButton.jsx

```diff
@@ -55,7 +54,7 @@ export class DeleteAccountButton extends Component {
         busy={deleting}
         onClick={this.handleDelete}
         label={t('accountForm.disconnect.button')}
-        {...omit(this.props, Object.keys(excludedButtonProptypes))}
+        {...omit(this.props, Object.keys(excludedButtonProptypes), 'client')}
```

Comment:
I don't think `omit` supports a  third argument? https://lodash.com/docs/4.17.15#omit

Reply:
It does 

```
require('lodash/omit')({ a: 1, b: 2}, ['a'], 'b')
```

The docs are not very clear but it is there ;)

> [paths] (...(string|string[])): The property paths to omit.

Reply:
TIL!